harperdb-connect.test.js

```diff
@@ -42,6 +43,7 @@ describe('HarperDBConnect Class', () => {
   })
```

Comment:
It turns out that this test got messed up when I extrapolated the `db = new HarperDBConnect()` into the `beforeEach()` method. This test should get updated to reinstantiate and expect that an instantiation with `username, password, and url` resolves with a reference to the instance of `HarperDBConnect` class. Give this a try, otherwise I'll make an independent issue.

Reply:
Ok, i'll try to update it in this PR.